### ISSUE TITLE
Query("#orddd_time_slot").selectWoo is not a function

### ISSUE THREAD

**Issue by kaji85** [TycheSoftwares/order-delivery-date-for-woocommerce#284]
After update of order delivery date for woocommerce lite, I have the following error message in the browser's console :+1: 
```
orddd-lite-initialize-datepicker.js?ver=3.11.2:13 Uncaught TypeError: jQuery(...).selectWoo is not a function
    at HTMLDocument.<anonymous> (orddd-lite-initialize-datepicker.js?ver=3.11.2:13)
    at i (jquery.js?ver=1.12.4-wp:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4-wp:2)
    at Function.ready (jquery.js?ver=1.12.4-wp:2)
    at HTMLDocument.J (jquery.js?ver=1.12.4-wp:2)
```
The datepicker is not show anymore and users cannot set a date with it. A customer finally set a date that was probably not in the right format as the date we see in the order is 1st January 1970.

I use the latest wordpress and woocommerce version, updated today for both.

I fixed the issue by commenting the line jQuery("#orddd_time_slot").selectWoo(); in orddd-lite-initialize-datepicker.js line 13, but I don't understand the initial issue. I activated the time slots and setup it but the error is still present.

I also tried to reproduce on a local wp instance, with same theme and some of the plugin of the main site and I have the same issue. 

If I can help on something, please tell me what to test.

Regards,
Benoit

**Comment by essaulsanchez** [comment]
Hi: 

I have the same problem, is there a solution?

Thanks!

Saúl

**Comment by kaji85** [comment]
Hi, 

As I said in my first message, I commented the line 13 of orddd-lite-initialize-datepicker.js, this is the only workaround I found. 

Regards,
Benoit

**Comment by rashmimalpande** [comment]
Hi @kaji85, @essaulsanchez ,

Can you give us some more details regarding this issue? Basically this function converts the normal select field into WooCommerce select2 field.

This could be a conflict with other plugins or theme on your site. You can check this once by switching the theme or deactivating the plugins.

Also, can you tell us if this issue was coming in version 3.11.1 also or it started with 3.11.2?

**Comment by kaji85** [comment]
Hello @rashmimalpande,

Thank you for your reply, I just tried some things : 
* desactivate all other plugins (except woocommerce of course and page builder) => does nothing
* I use the theme Bento by satori, I switched to storefront, and the issue disappeared immediately, I tried another theme (Twenty Seventeen) and I have no issue too. It seems that my issue comes from the Bento theme, that's a bit sad as I use this theme and don't want to switch now to another. And by the way, I don't even use the time slot functionnality.

I think the issue appears in 3.11.1 if my remember is right.

Regards,
Benoit
